Dockerfile

```diff
@@ -15,6 +15,7 @@ RUN \
   echo "**** install build packages ****" && \
   apk add --no-cache --virtual=build-dependencies \
     build-base \
+    cargo \
```

Comment:
Does Babybuddy actually build anything that needs Rust?

Reply:
I actually don't know. I know the main thing we usually want cargo for is cryptography, which is almost always an underlying dependency in the chain for most projects, but maybe it isn't here? Anyway I can remove it. It wasn't included before and seemed to build fine without it.